custom/conf/app.ini.sample

```diff
@@ -282,8 +282,9 @@ DISABLE_ROUTER_LOG = false
 ; not forget to export the private key):
 ; $ openssl pkcs12 -in cert.pfx -out cert.pem -nokeys
 ; $ openssl pkcs12 -in cert.pfx -out key.pem -nocerts -nodes
-CERT_FILE = custom/https/cert.pem
-KEY_FILE = custom/https/key.pem
+; Relative paths will be made absolute against the CUSTOM_PATH
```

Comment:
```suggestion
; Paths are relative to CUSTOM_PATH
```
Isn't the term "relative" actually? Absolute means from root. 🤔
Same in other parts of the PR.